Nos/Views/Moderation/UserFlagView.swift

```diff
@@ -0,0 +1,137 @@
+import SwiftUI
+
+/// Displays pickers for selecting user flag options, with additional stages shown
+/// based on previous selections.
+struct UserFlagView: View {
+    @Binding var selectedFlagOption: FlagOption?
+    @Binding var selectedSendOption: FlagOption?
+    @Binding var selectedVisibilityOption: FlagOption?
+    @Binding var showSuccessView: Bool
+
+    /// The target of the report.
+    let flagTarget: ReportTarget
+    var sendAction: () -> Void
```

Comment:
Oh, I'm curious what this does (though I think I have a good guess). Could you add a doc comment?

Reply:
Thanks for adding the doc comment!